pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -12,8 +12,10 @@
 , rustc
 , rustPlatform
 , rust-bindgen
+, nixosTests
+}:
 
-, # The kernel source tarball.
+{ # The kernel source tarball.
```

Comment:
It'd be great if these function args could be annotated to make it clear what they represent; one for deps and one for options.

It's clear to me now but it took a bit to understand.